server/bin/pbench-state-manager

```diff
@@ -0,0 +1 @@
+pbench-state-manager.py
```

Comment:
I originally wasn't going to bother with the link; I added it, in the end... but I forgot to add it to the spec file. Sigh.